watertap/examples/case_studies/municipal_treatment/municipal_treatment.py

```diff
@@ -0,0 +1,301 @@
+        "database": m.db})
+    m.fs.gravity_basin = FixedBedZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db,
+        "process_subtype": "gravity_basin"})
+    m.fs.gac = GACZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db,
+        "process_subtype": "pressure_vessel"})
+    m.fs.backwash_pump = WaterPumpingStationZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db,
+        "process_subtype": "treated"})
+    m.fs.uv = UVZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db})
+    m.fs.anion_exchange = IonExchangeZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db,
+        "process_subtype": "anion_exchange"})
+    m.fs.chlorination = ChlorinationZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db})
+    m.fs.storage = StorageTankZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db})
+    m.fs.recharge_pump = WaterPumpingStationZO(default={
+        "property_package": m.fs.prop,
+        "database": m.db,
+        "process_subtype": "treated"})
+
+    # connections
+    m.fs.s01 = Arc(source=m.fs.feed.outlet, destination=m.fs.intake_pump.inlet)
+    m.fs.s02 = Arc(source=m.fs.intake_pump.outlet, destination=m.fs.coag_and_floc.inlet)
+    m.fs.s03 = Arc(source=m.fs.coag_and_floc.outlet, destination=m.fs.sedimentation.inlet)
+    m.fs.s04 = Arc(source=m.fs.sedimentation.treated, destination=m.fs.ozonation.inlet)
+    m.fs.s05 = Arc(source=m.fs.ozonation.treated, destination=m.fs.gravity_basin.inlet)
+    m.fs.s06 = Arc(source=m.fs.gravity_basin.treated, destination=m.fs.gac.inlet)
+    m.fs.s07 = Arc(source=m.fs.gac.treated, destination=m.fs.uv.inlet)
+    m.fs.s08 = Arc(source=m.fs.gac.byproduct, destination=m.fs.backwash_pump.inlet)
+    m.fs.s09 = Arc(source=m.fs.uv.treated, destination=m.fs.anion_exchange.inlet)
+    m.fs.s10 = Arc(source=m.fs.anion_exchange.treated, destination=m.fs.chlorination.inlet)
+    m.fs.s11 = Arc(source=m.fs.chlorination.treated, destination=m.fs.storage.inlet)
+    m.fs.s12 = Arc(source=m.fs.storage.outlet, destination=m.fs.recharge_pump.inlet)
+    TransformationFactory("network.expand_arcs").apply_to(m)
+
+    # scaling
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def set_operating_conditions(m):
+    # ---specifications---
+    # feed
+    flow_vol = 0.9224 * pyunits.m**3/pyunits.s
+    conc_mass_tds = 0.63 * pyunits.kg/pyunits.m**3
+    conc_mass_tss = 0.006525 * pyunits.kg/pyunits.m**3
+    conc_mass_toc = 0.004 * pyunits.kg / pyunits.m ** 3
+
+    m.fs.feed.flow_vol[0].fix(flow_vol)
+    m.fs.feed.conc_mass_comp[0, "tds"].fix(conc_mass_tds)
+    m.fs.feed.conc_mass_comp[0, "tss"].fix(conc_mass_tss)
+    m.fs.feed.conc_mass_comp[0, "toc"].fix(conc_mass_toc)
+    solve(m.fs.feed)
+
+    # intake pump
+    m.db.get_unit_operation_parameters("water_pumping_station")
```

Comment:
Why are you calling `get_unit_operation_parameters` here? This is done automatically as part of `load_parameters_from_database` so all of these lines are unnecessary.

Reply:
Ok, I saw this before every `load_parameters_from_database()` in the tests, but I see that it was only to get the data object to do other things. The commercial seawater desalination case will need to be updated as well.